realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -594,11 +610,46 @@ public Builder schemaVersion(long schemaVersion) {
             return this;
         }
 
+        /**
+         * Update the schema version of the Realm and apply the provided migration.
+         * <p>
+         * The schema version must be equal to or higher than the schema version of the existing
```

Comment:
What happens if the version is less?

Reply:
The conventional `IllegalArgumentException` with message like `Provided schema version 1 is less than last set version 3`. I will update the docs.

Don't know if it would make sense to differentiate errors a bit. Also if changing scheme without bumping version number the migration is not triggered and we just emit the usual `RealmMigrationNeededException`.